src/features/authorSlice.ts

```diff
@@ -0,0 +1,24 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+
+type AuthorState = {
+  author: User | null;
+};
+
+const initialState: AuthorState = {
+  author: null,
+};
+
+const authorSlice = createSlice({
+  name: 'author',
+  initialState,
+  reducers: {
+    set: (state, actions: PayloadAction<User>) => ({
+      ...state,
+      author: actions.payload,
+    }),
```

Comment:
You can mutate your state here. RTK uses immer under the hood to make it immutable. Please, read the theory and watch videos carefully.